tests/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 Viam Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_executable(viam_cpp_sdk_tests
+  test_generic_component.cpp
+)
+
+target_link_libraries(viam_cpp_sdk_tests
+  viamcpp
+)
+
+install(
```

Comment:
Only necessary if you think installing the tests is useful and/or you want to test the installed test binaries rather than the ones from the build tree.